src/Symfony/Component/Form/ChoiceList/Factory/PropertyAccessDecorator.php

```diff
@@ -116,13 +114,13 @@ public function createListFromLoader(ChoiceLoaderInterface $loader, $value = nul
     /**
      * {@inheritdoc}
      *
-     * @param array|callable|string|PropertyPath|null $preferredChoices The preferred choices
-     * @param callable|string|PropertyPath|null       $label            The callable or path generating the choice labels
-     * @param callable|string|PropertyPath|null       $index            The callable or path generating the view indices
-     * @param callable|string|PropertyPath|null       $groupBy          The callable or path generating the group names
-     * @param array|callable|string|PropertyPath|null $attr             The callable or path generating the HTML attributes
+     * @param mixed $preferredChoices
+     * @param mixed $label
+     * @param mixed $index
+     * @param mixed $groupBy
+     * @param mixed $attr
      *
-     * @return ChoiceListView The choice list view
+     * @return ChoiceListView
      */
     public function createView(ChoiceListInterface $list, $preferredChoices = null, $label = null, $index = null, $groupBy = null, $attr = null)
```

Comment:
I mean using mixed for a decorator makes sense as the class does not know what it decorates that might have widened the range of accepted parameters types.
But in theory this applies to all arguments. So in theory you would also need to remove the type from `ChoiceListInterface $list` and accept mixed there. This might go to far. But it's a thought expirement.

Reply:
> This might go to far. But it's a thought experiment.

I agree with this :)